app/auth/utils.py

```diff
@@ -475,8 +482,10 @@ def _check_web_services_response(response, msg):
     Log an error message if the specified response's
     status code is not 200.
     """
-    if response.status_code != 200:
+    if response.status_code != 200 and response.json() is None:
```

Comment:
Why are we logging `response.json()` if it is `None`?

Reply:
Typo